tests/test_gitcharm.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+
+class TestGitCham(unittest.TestCase):
```

Comment:
I feel like I'm missing something, but this test case doesn't have any tests in it.